### PR TITLE
Assignment 3 & Assignment 4 OK

### PR DESCRIPTION
**Pull request by marioruiz2003**

### CODE REVIEW

**Comment by dgarijo** [comment]
The assignment looks ok, but you are not following the guidelines shown in class to upload it.
You have to create a new folder with your github id and student id, and then add the files there. Please change it

**Comment by ocorcho** [comment]
Assignment 3 ready to be accepted. @dgarijo can check assignment 4

File: Assignment4/task06_marioruiz2003_marioruiz.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/135dSjF_Z-KRec-YoL7KLcEIMPR-FchsT
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/task06_marioruiz2003_marioruiz.py:12]: this will not work in python, it's notebook magic

**Review by dgarijo — APPROVED** [review]

**Comment by marioruiz2003** [comment]
You have accepted my pull request ,how can I change what you have told me?
Should I create a new pull request with the folder you ask for?

**Comment by dgarijo** [comment]
Yes, create a new one

**Pull request merged**